config/project/project.yaml

```
@@ -50,13 +50,25 @@ email:
     username: $SYSTEM_EMAIL_USERNAME
   transportType: craft\mail\transportadapters\Smtp
 fs:
-  local:
+  s3:
     hasUrls: true
-    name: Local
+    name: S3
     settings:
-      path: '@webroot/assets'
-    type: craft\fs\Local
-    url: /assets/
+      addSubfolderToRootUrl: true
+      autoFocalPoint: false
+      bucket: $S3_BUCKET
+      bucketSelectionMode: manual
+      cfDistributionId: $S3_CLOUDFRONT_DISTRIBUTION_ID
+      cfPrefix: $S3_SUBFOLDER
+      expires: ''
```

I don't know if this is important or correct, but I see it on most of the other sites using this setup.

```suggestion
      expires: '30 days'
```